server/datastore/mysql/migrations/tables/20220714191431_AddHostOperatingSystemTable.go

```diff
@@ -0,0 +1,29 @@
+package tables
```

Comment:
Any reason to have two separate migration files? (you are not the first to do this :)

Having two separate migration files means they will end up as two entries in the `migration_status_tables` table.

Reply:
@lucasmrod, there wasn't a particular reason other than a slight preference on my part to test them separately. Would you like me to consolidate the two?

Reply:
I'd try to keep them in one file to signal that these go together (the feature needs both tables applied).